test/Vagrantfile

```diff
@@ -2,13 +2,4 @@ Vagrant.configure("2") do |config|
     config.vm.box = "local/hashistack"
     config.vm.network "private_network", ip: "10.0.3.10"
     config.vm.box_url = "file://../packer/output-hashistack/package.box"
-
-    config.vm.provision "ansible_local" do |startup|
-        run = "always"
-        startup.playbook = "/etc/ansible/startup.yml"
-    end
-    config.vm.provision "ansible_local" do |ansible|
-        ansible.provisioning_path = "/vagrant/ansible"
-        ansible.playbook = "playbook.yml"
-    end
```

Comment:
This is the test and should not be removed

Reply:
Oh oops, got the different playbooks mixed up. Thank you.

Reply:
if you see `playbook.yml` you know it it the startingpoint. Like a main class.

Reply:
Okay, that makes sense.

Reply:
Should be fixed now